### DISCUSSION TITLE
How to split bindings into two different source files?

### DISCUSSION THREAD

**Discussion by adamryczkowski** [wjakob/nanobind#650] [Q&A]
Hi!

For source tree clarity, I would like to split the file that defines Python bindings (which now holds the `NB_MODULE`) into two places. Is this allowed?

I am using nanobind with CMake.

**Comment by wojdyr** [comment]
Hi Adam,

yes, it's covered here: https://nanobind.readthedocs.io/en/latest/faq.html#how-can-i-reduce-build-time
